openstack/imageservice/v2/results.go

```diff
@@ -0,0 +1,349 @@
+package v2
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	
+	"github.com/rackspace/gophercloud"
+	//"github.com/rackspace/gophercloud/pagination"
+)
+
+// does not include the literal image data; just metadata.
+// returned by listing images, and by fetching a specific image.
+type Image struct {
+	Id string
+	
+	Name string
+	
+	Status ImageStatus
+	
+	Tags []string
+	
+	ContainerFormat *string
```

Comment:
@zaletniy It's a bit of a hack -- I was using it as an option type, to express the absence of a string. Unfortunately the `string` type in Go does not include `nil`. (Apparently it's idiomatic to use the empty string to mean "no string", but this seems inappropriate to me.)